controllers/resultController.js

```diff
@@ -0,0 +1,12 @@
+const { asyncHandler, responseHandler } = require('../middleware');
+const { resultModel } = require('../models');
+
+exports.getAllResults = asyncHandler(async (req, res, next) => {
+  const allResults = await resultModel.find().populate('user');
```

Comment:
how is the user able to take a peak at all tests of all other users!

shouldn't be there a condition inside the find to filter by the user?